.github/dependabot.yml

```diff
@@ -19,3 +19,11 @@ updates:
       - 'npm'
       - 'dependencies'
 
+  # Maintain dependencies for docker images in dockerfiles FROM clauses
+  - package-ecosystem: 'docker'
+    directory: '/'
+    schedule:
+      interval: 'daily'
+    labels:
+      - 'docker'
+      - 'dependencies'
```

Comment:
For now, we're probably going to stick with Node 16, as it's an LTS release. Can you add a rule to the dependabot file which prevents it from bumping to a version newer than Node 16? (Or at least, skips Node 17.)

It's a build-in feature in Dependabot:
https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates#ignore

I would like it to be something like this:

```suggestion
      - 'dependencies'
   ignore:
     - dependency-name: 'node'
        versions: ['version-update:semver-major']
```

However, I strongly doubt whether that will work, since the docker-label isn't strictly adhering to SemVer. I have no experience with limiting Dependabot versioning for Docker images, for npm/yarn it works fine, but there the ecosystem defines how to specify version ranges. 

Assuming the other suggestion doesn't work, it could instead be something like this:

```suggestion
      - 'dependencies'
   ignore:
     - dependency-name: 'node'
        versions: ['node:17-alpine']
```